### DISCUSSION TITLE
Regarding documentation.

### DISCUSSION THREAD

**Discussion by kimu-Sky** [treeder/material#25] [General]
I'm sorry but where is the documentation?

My projects are using [material-components/material-components-web](https://github.com/material-components/material-components-web/) and I want to migrate here because of the deprecation (pending) news.

**Comment by treeder** [comment]
Hi @kimu-Sky ,

The docs over there still apply: https://material-web.dev/ and you can look at the source of the demo page here too: https://treeder.github.io/material/demo/

For new components, I'm starting to add README's in the component directory for instance: https://github.com/treeder/material/tree/main/snackbar. 

Hopefully over time we can add them to all the components new and old.

**Reply by kimu-Sky** [reply]
Thanks very much for your effort!

When are you planning to add the [Top App Bar](https://m3.material.io/components/top-app-bar)?

**Reply by treeder** [reply]
Haven't started on it yet, but happy to take contributions.
